src/lib/components/particles/square-cut-corner-icon/index.jsx

```diff
@@ -0,0 +1,29 @@
+import defaultStyles from "./style.module.scss"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const SquareCutCornerIcon = ({cornerColor, squareColor, squareSize, cornerPartyAbbreviation, squarePartyAbbreviation, styles}) => {
+
+  styles = mergeStyles({...defaultStyles}, styles)
+
+  // expected size is 24px, with a corner size of 15px and a margin of 2px
+  let cornerSide = squareSize / 24 * 15;
+  let cornerMargin = squareSize < 100 ? squareSize / 12 : 10  // margin should increase with the size of square but cap at 10px
+
+    return (
+        <svg width={squareSize} height={squareSize} viewBox={`0 0 ${squareSize} ${squareSize}`} >
+
+            <g id="square-cutoff-corner-icon">
+
+              <rect id="square" fill={squareColor} className={`fill-color--${squarePartyAbbreviation}`} width={squareSize} height={squareSize} />
+
+                <g id="corner">
+
+                  <polygon id="cornerTriangleBg" className={styles.strokePrimaryBgColor} points={`0,0 0,${cornerSide} ${cornerSide},0`} />
```

Comment:
Is this a style that might need to be overridden? I wonder if we can come up with a name that betters describes the thing the style is applied to.

Reply:
No - it shouldn't need to be overridden. It's a dark mode adaption. ie if the background is white then the line dividing the corner from the square should be white so it looks like it's part of the background. Then if the background is black, the line should be black too. 

But I could call the style `dividingLineColor` or something like that